pkg/scheduler/plugins/reservation/plugin.go

```diff
@@ -236,6 +237,24 @@ func (p *Plugin) PostFilter(ctx context.Context, state *framework.CycleState, po
 		// return err to stop default preemption
 		return nil, framework.NewStatus(framework.Error)
 	}
+
+	nodeinfos, err := p.handle.SnapshotSharedLister().NodeInfos().List()
+	if err != nil {
+		return nil, framework.NewStatus(framework.Error)
+	}
+
+	p.parallelizeUntil(ctx, len(nodeinfos), func(piece int) {
+		nodeInfo := nodeinfos[piece]
+		for _, pi := range nodeInfo.Pods {
```

Comment:
how about finding the reservations via `(*AvailableCache).GetOnNode()` rather than visit all pods on the node

Reply:
good idea, thanks!